installer/charts/rhtap-acs-scanner/scripts/test-acs-image-scan.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+ROX_CENTRAL_ENDPOINT="$(kubectl get secrets -n rhtap rhtap-acs-integration -o jsonpath='{.data.endpoint}' | base64 -d)"
+ROX_API_TOKEN="$(kubectl get secrets -n rhtap rhtap-acs-integration -o jsonpath='{.data.token}' | base64 -d)"
+export ROX_CENTRAL_ENDPOINT
+export ROX_API_TOKEN
+
+IMAGE="${IMAGE:-quay.io/fedora/fedora:36-x86_64@sha256:d6e4c7d6d1eaa24d71c8efd7432890acdc0179502224d0aaad6bb05d15ffde19}"
+
+echo "# Download roxctl cli from ${ROX_CENTRAL_ENDPOINT}"
+curl --fail --insecure -s -L -H "Authorization: Bearer $ROX_API_TOKEN" \
+  "https://${ROX_CENTRAL_ENDPOINT}/api/cli/download/roxctl-linux" \
+  --output ./roxctl  \
+  > /dev/null || {
+    echo '[ERROR] Failed to download roxctl'
+    sleep 600
+    exit 1
+  }
+chmod +x ./roxctl  > /dev/null
+echo
+
+while true; do
+  echo "# roxctl image scan"
+  date
+  if ./roxctl image scan \
+      "--insecure-skip-tls-verify" \
+      -e "${ROX_CENTRAL_ENDPOINT}" \
+      --image "$IMAGE" \
+      --output json \
+      --force; then
+    break
+  fi
+  echo "Waiting"
+  echo
+  sleep 60
+  echo "Retrying"
+done
+rm ./roxctl
+echo
```

Comment:
Please use a `emptyDir: {}` mount, so at the end of the container execution all data in the mount point will be gone.

Reply:
Fixed